geocoding/example/android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,11 +1,11 @@
 arguments=
 auto.sync=false
 build.scans.enabled=false
-connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.gradle.distribution=GRADLE_DISTRIBUTION(VERSION(7.1.1))
 connection.project.dir=
 eclipse.preferences.version=1
 gradle.user.home=
-java.home=/Users/maurits/Library/Java/JavaVirtualMachines/openjdk-14.0.1/Contents/Home
+java.home=C\:/Program Files/Java/jdk-18.0.1.1
```

Comment:
Can you revert this change or don't check it in

Reply:
This did not resolve correctly 🤔 @Mayb3Nots

Reply:
My bad